sklearn/decomposition/online_lda.py

```diff
@@ -246,7 +246,7 @@ class LatentDirichletAllocation(BaseEstimator, TransformerMixin):
     """
 
     def __init__(self, n_topics=10, doc_topic_prior=None,
-                 topic_word_prior=None, learning_method='online',
+                 topic_word_prior=None, learning_method='batch',
```

Comment:
this an API change. You cannot do this without a proper deprecation cycle. If it's what "people" want